src/ImpactEvaluator.sol

```diff
@@ -106,6 +107,14 @@ contract ImpactEvaluator is AccessControl {
         reward(addresses, scores);
     }
 
+    function validateScores(uint64[] memory scores) pure private {
+        uint64 sum = 0;
+        for (uint i = 0; i < scores.length; i++) {
+            sum += scores[i];
+        }
+        require(sum <= 1e15, "Sum of scores too big");
```

Comment:
Can we extract this magic number `1e15` into a named constant please?
1. Make it easier to understand what this number means.
2. Ensure we use the same value in all places (validate, reward)

Reply:
already done in https://github.com/Meridian-IE/impact-evaluator/commit/5fff3f73669b6ae8aed139a979699f8d1d11e463 🙏

Reply:
@juliangruber I think you forgot to update this line to use that constant. Could you PTAL?

This is what I see on the current main branch:
https://github.com/Meridian-IE/impact-evaluator/blob/7399cc79c96579ef901c7855e921ddd0fb407c1b/src/ImpactEvaluator.sol#L117-L123

Reply:
good catch! fixed in https://github.com/Meridian-IE/impact-evaluator/commit/d735a12ac65f1c59b1b8b9a84030ede364175724